### PR TITLE
needed to include the depends on python cookbook

### PR DESCRIPTION
**Pull request by ddegoede**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -8,3 +8,5 @@
 
 supports         'redhat'
 supports         'centos'
+
+depends          'python', '= 1.4.6'
```
Comment (svanharmelen) [inline metadata.rb:12]: Is there a need to add this hard dependency? Or could we do with something like `'python', '> 1.4.0'` (or whatever version is the minimal needed version)?

And... I noticed this line in the cookbook readme: `THIS COOKBOOK IS DEPRECATED` adn it says the better cookbook to use is https://github.com/poise/poise-python so I suggest we do that right?

**Comment by hrak** [comment]
`~> 1.4.6` will probably do fine for now. At the moment this breaks the cookbook, chef throws an error

```
  Chef::Exceptions::CookbookNotFound
  ----------------------------------
  Cookbook python not found. If you're loading python from another cookbook, make sure you configure the dependency in your metadata
```

**Comment by hrak** [comment]
This works for me, tested on Ubuntu 14.04 & Chef 12.5.1.

``` diff
diff --git a/metadata.rb b/metadata.rb
index a791300..c8bde78 100755
--- a/metadata.rb
+++ b/metadata.rb
@@ -6,5 +6,8 @@ description      'Installs/Configures cloudmonkey'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.1.1'

-supports         'redhat'
-supports         'centos'
+%w(debian centos redhat ubuntu).each do |os|
+  supports os
+end
+
+depends          'python', '~> 1.4.6'
```

**Comment by sspans** [comment]
I've created pull #3 to resolve the issues raised here.

**Pull request closed without merging**